recipes/nibabel/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
Also add the arguments here too.

     --single-version-externally-managed --record record.txt